keylime/policy/create_runtime_policy.py

```diff
@@ -434,6 +434,13 @@ def get_arg_parser(create_parser: _SubparserType, parent_parser: argparse.Argume
             dest="remote_rpm_repo",
             help="Remote RPM repo URL",
         )
+        repo_group.add_argument(
+            "--max-workers",
+            dest="max_workers",
+            type=_positive_int,
+            default=rpm_repo.MAX_WORKERS_DEFAULT,
+            help="The maximum number of workers used for RPM processing (must be a positive integer).",
```

Comment:
Recommendation: Update the help text to include the default, which is a common and helpful practice for command-line tools. For example:
```help="The maximum number of workers used for RPM processing (must be a positive integer; default: 8)."```

Reply:
Thank you, updated.